x-pack/metricbeat/module/iis/website/manifest.yml

```diff
@@ -5,57 +5,86 @@ input:
   defaults:
     perfmon.group_measurements_by_instance: true
     perfmon.ignore_non_existent_counters: true
-    perfmon.counters:
-    #network
-    - instance_label: 'name'
-      measurement_label: total_bytes_received
-      query: '\Web Service(*)\Total Bytes Received'
-    - instance_label: 'name'
-      measurement_label: total_bytes_sent
-      query: '\Web Service(*)\Total Bytes Sent'
-    - instance_label: 'name'
-      measurement_label: bytes_sent_per_sec
-      query: '\Web Service(*)\Bytes Sent/sec'
-    - instance_label: 'name'
-      measurement_label: bytes_received_per_sec
-      query: '\Web Service(*)\Bytes Received/sec'
-    - instance_label: 'name'
-      measurement_label: current_connections
-      query: '\Web Service(*)\Current Connections'
-    - instance_label: 'name'
-      measurement_label: maximum_connections
-      query: '\Web Service(*)\Maximum Connections'
-    - instance_label: 'name'
-      measurement_label: total_connection_attempts
-      query: '\Web Service(*)\Total Connection Attempts (all instances)'
-    - instance_label: 'name'
-      measurement_label: total_get_requests
-      query: '\Web Service(*)\Total Get Requests'
-    - instance_label: 'name'
-      measurement_label: get_requests_per_sec
-      query: '\Web Service(*)\Get Requests/sec'
-    - instance_label: 'name'
-      measurement_label: total_post_requests
-      query: '\Web Service(*)\Total Post Requests'
-    - instance_label: 'name'
-      measurement_label: post_requests_per_sec
-      query: '\Web Service(*)\Post Requests/sec'
-    - instance_label: 'name'
-      measurement_label: total_delete_requests
-      query: '\Web Service(*)\Total Delete Requests'
-    - instance_label: 'name'
-      measurement_label: delete_requests_per_sec
-      query: '\Web Service(*)\Delete Requests/sec'
-    - instance_label: 'name'
-      measurement_label: service_uptime
-      query: '\Web Service(*)\Service Uptime'
-    - instance_label: 'name'
-      measurement_label: total_put_requests
-      query: '\Web Service(*)\Total PUT Requests'
-    - instance_label: 'name'
-      measurement_label: put_requests_per_sec
-      query: '\Web Service(*)\PUT Requests/sec'
+    perfmon.queries:
+    - object: 'Web Service'
+      instance: "*"
+      counters:
+      - name: 'Total Bytes Received'
+        field: total_bytes_received
```

Comment:
what would be the resulting field name if you don't add this line?

Reply:
`total_bytes_received` good point, could remove the extra `field` options